lib/oneview-sdk/resource.rb

```diff
@@ -323,6 +323,15 @@ def ensure_uri
       true
     end
 
+    # Fail unless all uri of resources is set.
```

Comment:
I recommend to improve this comment:
1. First it would be better if it was a description of what it really does. (**e.g.** *Gets all the URIs for the specified resources*)
2. Add the YARD tags for the @param, @return and @raise